src/app/course/_forms/course-event.form.ts

```diff
@@ -32,7 +32,7 @@ export interface EventQuestionSetFormData {
 
 export class CourseEventForm {
     /**
-     * Creates a FormGroup for a Course Event.
+     * Creates a FormGroup for a Course Assessment.
```

Comment:
here `Course Event` refers to out model `CourseEvent`, therefore should not be changed.